### PR TITLE
Use types and named constants from nupic.data.fieldmeta in aggregator module instead of naked constants.

### PR DESCRIPTION
**Pull request by vitaly-krugl**
CC @scottpurdy 

This is a simple change. Unfortunately, the IDE eliminated a bunch of whitespace, so it looks like there were a lot of mods. To make it easy to review, I tagged my changes with github comments: change 1, change 2, change 3.

### CODE REVIEW

File: src/nupic/data/aggregator.py [hunk 1]
```diff
@@ -20,14 +20,14 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-import os
-import time
-import datetime
 from collections import defaultdict
-
+import datetime
```
Comment (vitaly-krugl) [inline src/nupic/data/aggregator.py]: Here, I simply rearranged the imports to be in sorted order (unrelated to the core of the change)

File: src/nupic/data/aggregator.py [hunk 2]
```diff
@@ -273,65 +273,63 @@ def __init__(self, aggregationInfo, inputFields, timeFieldName=None,
...
-  
-    - inputFields: The fields from the source. This is a list of tuples as
-          returned from the 'fields' member of a RecordStreamIFace. The tuples
-          are: (fieldName, type, special). The type can be 'float', 'int', 
-          'string', 'datetime' or 'bool'. And the special can be '', 
-          'T' (for timestamp), 'S' (for sequenceId), or 'R' (for reset).  
-  
+
+    - inputFields: The fields from the data source. This is a sequence of
+      `nupic.data.fieldmeta.FieldMetaInfo` instances.
```
Comment (vitaly-krugl) [inline src/nupic/data/aggregator.py]: core change 1

File: src/nupic/data/aggregator.py [hunk 3]
```diff
@@ -273,65 +273,63 @@ def __init__(self, aggregationInfo, inputFields, timeFieldName=None,
...
     If the input file contains a time field, sequence id field or reset field
     that were not specified in aggregationInfo fields, those fields will be
     added automatically with the following rules:
-    
+
     1. The order will be R, S, T, rest of the fields
-    2. The aggregation function for all will be to pick the first: lambda x: x[0]
-    
+    2. The aggregation function for these will be to pick the first:
+       lambda x: x[0]
```
Comment (vitaly-krugl) [inline src/nupic/data/aggregator.py]: change 2

File: src/nupic/data/aggregator.py [hunk 4]
```diff
@@ -392,24 +390,25 @@ def __init__(self, aggregationInfo, inputFields, timeFieldName=None,
...
+
         # Is it a special field that we are still looking for?
-        if special == 'R' and self._resetFieldIdx is None:
+        if special == FieldMetaSpecial.reset and self._resetFieldIdx is None:
           self._resetFieldIdx = fieldIdx
-        if special == 'T' and self._timeFieldIdx is None:
+        if special == FieldMetaSpecial.timestamp and self._timeFieldIdx is None:
           self._timeFieldIdx = fieldIdx
-        if special == 'S' and self._sequenceIdFieldIdx is None:
+        if (special == FieldMetaSpecial.sequence and
```
Comment (vitaly-krugl) [inline src/nupic/data/aggregator.py]: change 3

**Comment by scottpurdy** [comment]
:+1:

**Pull request merged**